samples/linq/csharp/aggregate/program.cs

```diff
@@ -0,0 +1,12 @@
+public class Program
+    {
+        public void Main(string[] args)
+        {
+            var sample1 = new Aggregate.AggregateSample1();
```

Comment:
Two things:

1. Would probably look cleaner to `using Aggregate;` at the top rather than use the fully-qualified namespace.
2. Why instance methods?  If these are conceptually just functions we call, what is the benefit of making them instance methods instead of static methods?